src/wow.coffee

```diff
@@ -152,6 +173,14 @@ class @WOW
     @applyStyle(box)
     box.className = "#{box.className} #{@config.animateClass}"
     @config.callback(box) if @config.callback?
+    @util().emitEvent(box, @wowEvent)
+
+    @util().addEvent(box, 'animationend', @resetAnimation)
+    @util().addEvent(box, 'oanimationend', @resetAnimation)
+    @util().addEvent(box, 'webkitAnimationEnd', @resetAnimation)
+    @util().addEvent(box, 'MSAnimationEnd', @resetAnimation)
```

Comment:
Can you please link to a reference or explain why you need the `webkit` and `MS` vendor prefixes, but no `moz` and `o`?